src/btwa_api/scripts.py

```diff
@@ -10,3 +11,20 @@ def add_handler(username, password):
     session.add(handler)
     session.commit()
     session.close()
+
+
+@click.command("add-handler")
+@click.argument("username")
+@click.argument("password")
```

Comment:
consider ask for password `getpass.getpass` (does not leave traces in history files) or generate password instead (might also implement basic password policy)

Reply:
Good point. This is just a script for local debugging, but I added it nevertheless.

Reply:
resolved by 750541d